src/main/java/net/snowflake/client/core/QueryContextDTO.java

```diff
@@ -0,0 +1,36 @@
+package net.snowflake.client.core;
+import java.util.List;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class QueryContextDTO {
+
+    private QueryContextEntryDTO mainEntry;
```

Comment:
Why do we have both a mainEntry and a list of entries v.s. just a list of entries?

Reply:
Yeah. I can update the `QueryContextDTO` to a list of entries.